test/tasks.e2e-spec.ts

```diff
@@ -0,0 +1,221 @@
+import { TaskStatus } from './../src/tasks/models/task.entity';
+import { ExecutionContext, INestApplication, UnauthorizedException, ValidationPipe } from '@nestjs/common';
+import { ContextIdFactory } from '@nestjs/core';
+import { Test, TestingModule } from '@nestjs/testing';
+import { getModelForClass, mongoose, ReturnModelType } from '@typegoose/typegoose';
+import * as request from 'supertest';
+
+import { JwtAuthGuard } from '../src/auth/guards';
+import { JwtPayload } from '../src/auth/models/jwt-payload';
+import { JwtStrategy } from '../src/auth/strategies/jwt.strategy';
+import { Stage } from '../src/stages/models/stage.entity.ts';
+import { AssignTasksDto } from '../src/tasks/models/dtos/assign-tasks.dto';
+import { CreateTaskDto } from '../src/tasks/models/dtos/create-task.dto';
+import { Task } from '../src/tasks/models/task.entity';
+import { TasksModule } from '../src/tasks/tasks.module';
+import { TasksService } from '../src/tasks/tasks.service';
+import { Track } from '../src/tracks/models/track.entity';
+import { CreateUserDto } from '../src/users/models/dto/create-user.dto';
```

Comment:
Codacy found an issue: ['CreateUserDto' is defined but never used.](https://app.codacy.com/gh/codeclannigeria/codeclannigeria-backend/pullRequest?prid=5781977)